src/clj/rems/email/core.clj

```diff
@@ -36,8 +36,20 @@
 (defn generate-handler-reminder-emails! []
   (doseq [email (->> (workflow/get-handlers)
                      (map (fn [handler]
-                            (let [lang (:language (user-settings/get-user-settings (:userid handler)))]
-                              (template/handler-reminder-email lang handler (todos/get-todos (:userid handler))))))
+                            (let [lang (:language (user-settings/get-user-settings (:userid handler)))
+                                  apps (todos/get-todos (:userid handler))]
+                              (template/handler-reminder-email lang handler apps))))
+                     (remove nil?))]
+    (enqueue-email! email)))
+
+(defn generate-reviewer-reminder-emails! []
+  (doseq [email (->> (applications/get-users-with-role :commenter) ; TODO: rename commenter to reviewer
+                     (map users/get-user)
+                     (map (fn [reviewer]
+                            (let [lang (:language (user-settings/get-user-settings (:userid reviewer)))
+                                  apps (->> (todos/get-todos (:userid reviewer))
+                                            (map #(= :waiting-for-your-review (:application/todo %))))]
```

Comment:
How about :waiting-for-your-decision, should it be included here, as well?

Reply:
Maybe, but nobody has yet asked for an email reminder for deciders. Some sort of deduplication could then be needed.